src/app-layout/skeleton/index.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import { AppLayoutPropsWithDefaults } from '../interfaces';
+import { SkeletonLayout } from './layout';
+
+export function AppLayoutSkeleton({
+  notifications,
+  contentHeader,
+  content,
+
+  navigation,
+  navigationHide,
+  navigationOpen,
+  navigationWidth,
+  tools,
+  toolsHide,
+  toolsOpen,
+  toolsWidth,
+
+  placement,
+  contentType,
+  maxContentWidth,
+  minContentWidth,
+  disableContentPaddings,
+}: AppLayoutPropsWithDefaults) {
+  // render nothing in the skeleton state
+  const placeholder = <></>;
```

Comment:
There are some plans to render some kind of grey boxes, but let's keep an empty fragment for now